src/test/lib/ApiTest.cpp

```diff
@@ -2831,6 +2844,30 @@ void QuicTestGlobalParam()
         for (uint32_t i = 0; i < ARRAYSIZE(Expected); ++i) {
             TEST_EQUAL(Sizes[i], Expected[i]);
         }
+
+        //
+        // Partial retrieve
+        //
+        Length = sizeof(uint32_t);
+        TEST_QUIC_SUCCEEDED(
+            MsQuic->GetParam(
+                nullptr,
+                QUIC_PARAM_GLOBAL_STATISTICS_V2_SIZES,
+                &Length,
+                Sizes));
```

Comment:
nit: It'd be even better to pass a pointer to a single uint32 here, to catch an overflow in the copy (and not only the value of `Length` being too big).